scripts/kubestellar

```diff
@@ -243,6 +244,24 @@ EOF
     echo "Finished augmenting root:compute for KubeStellar"
 }
 
+function ensure_space_mgt() {
+    kubectl ws root &> /dev/null
+    $SCRIPT_DIR/space-provider-config "$KUBECONFIG" "$SCRIPT_DIR/../config/space/provider_kcp.yaml"
+    if kubectl get Workspace "$space_mgt_name" &> /dev/null; then
+	echo "space-mgt workspace already exists -- using it:"
+	kubectl ws "$space_mgt_name"
+    else
+        kubectl ws create "$space_mgt_name" --enter
+    fi
+    kubectl ws create-context --overwrite
+    # create space CRDs
+    eval kubectl apply -f "$SCRIPT_DIR/../config/crds/space.kubestellar.io_spaces.yaml" $verbdir
```

Comment:
Look at the picture of software components in https://docs.google.com/drawings/d/1U5nwzHeSdpEufzOPmYX_8cIvNOUzRa3oOFuKqNS4smM .

If we put multiple components in one GitHub _repository_ then we should cleanly separate them at a high level in the directory structure.  We are slouching towards a monorepo.  We can no longer afford to slouch, this needs to be done with some recognition and discipline.